apiserver/plane/api/views/issue_type.py

```
@@ -0,0 +1,240 @@
+            #     current_instance=None,
+            #     epoch=int(timezone.now().timestamp()),
+            # )
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def patch(self, request, slug, project_id, issue_id, pk):
+        issue_type = IssueType.objects.get(
+            workspace__slug=slug,
+            pk=pk,
+        )
+        requested_data = json.dumps(self.request.data, cls=DjangoJSONEncoder)
```

Ruff (0.8.2)

Local variable `requested_data` is assigned to but never used

Remove assignment to unused variable `requested_data`

(F841)